src/utils/helpers.tsx

```diff
@@ -0,0 +1,23 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const prepareTodos = (
+  todosList: Todo[],
+  selectedFilter: Filter,
+): Todo[] => {
+  let filteredTodos = [...todosList];
```

Comment:
The `filter` array method returns a copy of the new array, so you don't have to do it manually